src/open_inwoner/haalcentraal/tests/test_client.py

```diff
@@ -0,0 +1,21 @@
+from django.test import TestCase
+
+import requests_mock
+
+from ..api import BRP_2_1
+from .mixins import HaalCentraalMixin
+
+
+@requests_mock.Mocker()
+class BRPClientTest(HaalCentraalMixin, TestCase):
+    def test_request_content_type(self, m):
```

Comment:
I think this test should also check if the request in `m.request_history` has the `Content: application/json` request header, because now we're just testing if the mock returns the right response.